app/src/main/res/layout/chat_massage.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do we need this file? looks like relevant for the chat that we dropped.

Reply:
right!